### DISCUSSION TITLE
leetcode-js/react/3

### DISCUSSION THREAD

**Discussion by giscus** [2xiao/leetcode-js#11] [Announcements]
# leetcode-js/react/3

3. 实现 Reconciler 摘要 Reconciler 简介 实现 FiberNode 实现 Reconciler 工作流程 相关代码可在 git tag v1.3 查看

https://2xiao.github.io/leetcode-js/react/3.html

**Comment by XC0703** [comment]
想问下流程图是用什么做的

**Reply by 2xiao** [reply]
搞定设计
